shopify/pre-purchase-template/web/routes/index.tsx

```diff
@@ -59,14 +67,14 @@ const PrePurchaseForm = ({ products, shop }) => {
           <Controller
             name="shopifyShop.prePurchaseProduct"
             control={control}
-            required
             render={({ field: { ref, ...fieldProps } }) => (
               <Select
                 label="Product for pre-purchase offer"
                 placeholder="-No product selected-"
                 options={products}
                 disabled={formState.isSubmitting}
                 {...fieldProps}
+                value={fieldProps.value ?? ""}
```

Comment:
This is annoying - do we need to do this for all Select inputs?

Reply:
I couldn't find another way to make it work. I might just not be doing it right

Reply:
Link to ticket: https://linear.app/gadget-dev/issue/GGT-7368/inconsistent-typing-between-useactionform-and-api-client-and-neither